game-engine/game-rules.md

```diff
@@ -299,10 +299,15 @@ Amount of damage suffered per object collision
 * Wall: 2
 * Cybertruck: 2
 
+How damage affects your current speed
```

Comment:
I think just to be more explicit can we have a list like below:
if damage is 6 then max speed is 0
if damage is 5 then max speed is 3
if damage is 4 then max speed is 6
if damage is 3 then max speed is 8
etc